applications/GeoMechanicsApplication/tests/cpp_tests/test_calculate_incremental_displacement_process.cpp

```diff
@@ -0,0 +1,63 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Aron Noordam
+//
+
+#include "custom_processes/calculate_incremental_displacement_process.h"
+#include "geo_mechanics_application_variables.h"
+#include "geo_mechanics_fast_suite.h"
+
+namespace Kratos::Testing
+{
+
+using namespace Kratos;
+
+KRATOS_TEST_CASE_IN_SUITE(CalculateIncrementalDisplacementProcess, KratosGeoMechanicsFastSuite)
+{
+    // set up model part
+    Model model        = Model();
+    auto& r_model_part = model.CreateModelPart("dummy", 2);
+    r_model_part.AddNodalSolutionStepVariable(DISPLACEMENT);
+    r_model_part.AddNodalSolutionStepVariable(INCREMENTAL_DISPLACEMENT);
+
+    auto p_node_1 = r_model_part.CreateNewNode(0, 0.0, 0.0, 0.0);
+    auto p_node_2 = r_model_part.CreateNewNode(1, 0.0, 0.0, 0.0);
+
+    std::vector<intrusive_ptr<Node>> nodes = {p_node_1, p_node_2};
+
+    for (auto p_node : nodes) {
+        p_node->AddDof(DISPLACEMENT_X);
+        p_node->AddDof(DISPLACEMENT_Y);
+        p_node->AddDof(DISPLACEMENT_Z);
+    }
+
+    // set up displacement values
+    p_node_1->FastGetSolutionStepValue(DISPLACEMENT, 0) = Kratos::array_1d<double, 3>{1.0, 2.0, 3.0};
+    p_node_1->FastGetSolutionStepValue(DISPLACEMENT, 1) = Kratos::array_1d<double, 3>{4.0, 5.0, 6.0};
+
+    p_node_2->FastGetSolutionStepValue(DISPLACEMENT, 0) = Kratos::array_1d<double, 3>{7.0, 12.0, 9.0};
+    p_node_2->FastGetSolutionStepValue(DISPLACEMENT, 1) = Kratos::array_1d<double, 3>{11.0, 8.0, 14.0};
+
+    // initialize process to be tested
+    CalculateIncrementalDisplacementProcess process =
+        CalculateIncrementalDisplacementProcess(r_model_part, Parameters(R"({})"));
```

Comment:
There is no need to repeat the type (and default-constructing a `Parameters` object should be sufficient):
```suggestion
    auto process = CalculateIncrementalDisplacementProcess(r_model_part, {});
```